server/client.go

```diff
@@ -856,6 +856,15 @@ func (c *client) flushOutbound() bool {
 					// here, and don't report a slow consumer error.
 					sce = false
 				}
+			} else {
+				// Under some conditions, a client may hit a slow consumer write deadline
+				// before the authorization or TLS handshake timeout. If that is the case,
+				// then we handle as slow consumer though we do not increase the counter
+				// as can be misleading.
+				if !c.flags.isSet(connectReceived) {
```

Comment:
If the only conditional, can we just make this an else if block with above?

Reply:
yes only conditional so far so sounds good